csv.go

```diff
@@ -50,17 +50,26 @@ func NewCSVReader(r io.Reader, d rune) *csv.Reader {
 	return cr
 }
 
-func CSVTable(cr *csv.Reader, key []string) (Table, error) {
+func CSVTable(cr *csv.Reader, key []string, renames map[string]string) (Table, error) {
 	cols, err := cr.Read()
 	if err != nil {
 		return nil, err
 	}
 
+	for i, k := range key {
+		if newName, ok := renames[k]; ok {
+			key[i] = newName
```

Comment:
Another nit. It is not idiomatic to have verbose names like `newName` in such a tiny block/scope. Change to `n` or something small (applies to other all similar code).